### DISCUSSION TITLE
Is there a way to always show a key for chart colors?

### DISCUSSION THREAD

**Discussion by kculmback-eig** [TanStack/react-charts#276] [Q&A]
We would like to render a key that is always visible that displays the color associated with each of the series on the secondary axis. Similar to what the tooltip shows but it is always visible and wouldn't have any counts.

Is this doable with the current API?

**Comment by RichardNolan** [comment]
I pass the defaultColors to the chart and then map over the data series using the label for display and the index to determine the color.

![image](https://user-images.githubusercontent.com/3406093/177760979-b9bd9dd1-e25b-4ef6-a578-e19cf970c0f7.png)
